File: commons-aws/src/test/java/test/pl/chilldev/commons/aws/GenericMessage.java
/**
 * This file is part of the ChillDev-Commons.
 *
 * @license http://mit-license.org/ The MIT license
 * @copyright 2018 © by Rafał Wrzeszcz - Wrzasq.pl.
 */

package test.pl.chilldev.commons.aws;

import java.util.List;
import java.util.UUID;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class GenericMessage
{
    private List<UUID> ids;
}

File: commons-aws/src/test/kotlin/test/pl/wrzasq/commons/aws/messaging/GenericMessage.kt
/*
 * This file is part of the pl.wrzasq.commons.
 *
 * @license http://mit-license.org/ The MIT license
 * @copyright 2018 - 2019, 2021 © by Rafał Wrzeszcz - Wrzasq.pl.
 */

package test.pl.wrzasq.commons.aws.messaging

import java.util.UUID

data class GenericMessage(
    var ids: List<UUID>
)
